### PR TITLE
Create adafruitio_led_SSL_esp8266.ino

### PR DESCRIPTION
**Pull request by aandroide**
Thank you for creating a pull request to contribute to Adafruit's GitHub code!
Before you open the request please review the following guidelines and tips to
help it be more easily integrated:

- **Describe the scope of your change--i.e. what the change does and what parts
  of the code were modified.**  This will help us understand any risks of integrating
  the code.

- **Describe any known limitations with your change.**  For example if the change
  doesn't apply to a supported platform of the library please mention it.

- **Please run any tests or examples that can exercise your modified code.**  We
  strive to not break users of the code and running tests/examples helps with this
  process.

Thank you again for contributing!  We will try to test and integrate the change
as soon as we can, but be aware we have many GitHub repositories to manage and
can't immediately respond to every request.  There is no need to bump or check in
on a pull request (it will clutter the discussion of the request).

Also don't be worried if the request is closed or not integrated--sometimes the
priorities of Adafruit's GitHub code (education, ease of use) might not match the
priorities of the pull request.  Don't fret, the open source community thrives on
forks and GitHub makes it easy to keep your changes in a forked repo.

After reviewing the guidelines above you can delete this text from the pull request.

### CODE REVIEW

File: esempi/adafruitio_led_SSL_esp8266.ino [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+  Written by Tony DiCola for Adafruit Industries.
+  MIT license, all text above must be included in any redistribution
+ ****************************************************/
+#include <ESP8266WiFi.h>
+#include "Adafruit_MQTT.h"
+#include "Adafruit_MQTT_Client.h"
+
+// the on off button feed turns this LED on/off
+#define LED D6  //example pin for d1 mini
```
Comment (brentru) [inline esempi/adafruitio_led_SSL_esp8266.ino]: Can you change this `#define` to pin 13? The on-board LED pin for the Feather ESP8266, most people use this sketch with that board.

File: esempi/adafruitio_led_SSL_esp8266.ino [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+  Serial.println("WiFi connected");
+  Serial.println("IP address: "); Serial.println(WiFi.localIP());
+  
+  // Setup MQTT subscription for onoff & slider feed.
+  mqtt.subscribe(&onoffbutton);
+  // check the fingerprint of io.adafruit.com's SSL cert
+  verifyFingerprint();
+ }
+
+uint32_t x=0;
```
Comment (brentru) [inline esempi/adafruitio_led_SSL_esp8266.ino]: What is this `x` variable being used for? Leftover code from previous example?

File: esempi/adafruitio_led_SSL_esp8266.ino [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+  MQTT_connect();
+
+  // this is our 'wait for incoming subscription packets' busy subloop
+  // try to spend your time here
+
+  Adafruit_MQTT_Subscribe *subscription;
+  while ((subscription = mqtt.readSubscription(5000))) {
+    // Check if its the onoff button feed
+    if (subscription == &onoffbutton) {
+      Serial.print(F("On-Off button: "));
```
Comment (brentru) [inline esempi/adafruitio_led_SSL_esp8266.ino]: Change `On-Off button: ` to `Button: ` so the serial monitor will print something like `Button: ON`

File: esempi/adafruitio_led_SSL_esp8266.ino [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+
+void verifyFingerprint() {
+
+  const char* host = AIO_SERVER;
+
+  Serial.print("Connecting to ");
+  Serial.println(host);
+
+  Serial.printf("Using fingerprint '%s'\n", fingerprint);
+  client.setFingerprint(fingerprint);
```
Comment (brentru) [inline esempi/adafruitio_led_SSL_esp8266.ino]: Please remove Line 131/132 - we'll check against the fingerprint, we don't need to set it.

File: esempi/adafruitio_led_SSL_esp8266.ino [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+
+  Works great with Adafruit's Huzzah ESP board & Feather
+  ----> https://www.adafruit.com/product/2471
+  ----> https://www.adafruit.com/products/2821
+
+  Adafruit invests time and resources providing this open source code,
+  please support Adafruit and open-source hardware by purchasing
+  products from Adafruit!
+
+  Written by Tony DiCola for Adafruit Industries.
```
Comment (brentru) [inline esempi/adafruitio_led_SSL_esp8266.ino]: Would you like to add your username here?

File: esempi/adafruitio_led_SSL_esp8266.ino [hunk 6]
```diff
@@ -0,0 +1,173 @@
+/***************************************************
+  Adafruit MQTT Library ESP8266 Example
```
Comment (brentru) [inline esempi/adafruitio_led_SSL_esp8266.ino]: Example needs to be renamed.

**Comment by brentru** [comment]
Travis also built with errors (visible here: https://travis-ci.com/adafruit/Adafruit_MQTT_Library/builds/101525828#L663). These'll need to be resolved before merging, too.

File: esempi/adafruitio_led_SSL_esp8266.ino (hunk 4 above)
Comment (aandroide) [reply esempi/adafruitio_led_SSL_esp8266.ino]: removing this, it will not work

Comment (aandroide) [reply esempi/adafruitio_led_SSL_esp8266.ino]: the module resets as soon as connected

**Pull request closed without merging**